data/human/humanity interludes.txt

```diff
@@ -0,0 +1,192 @@
+			`	He flips through a few more slides, these ones show the broken husks of Sestor ships, wrecks, and some salvaged outfits spread out on a tarmac. "As a result, Parliament has seen fit to significantly increase the Navy's R&D budget. This, combined with the salvage and debris we have collected from theses 'Sestor' ships has allowed several of our long-standing projects to accelerate." He turns towards you. "This is where you come in: You have more combat experience against the Sestor than any other captain we know of. We would like you to inspect a new weapon we have developed, observe a weapons demonstration test, and then give us your thoughts on how effective it will be against the Sestor. So, do you accept?"`
+			choice
+				`	"I accept."`
+					goto briefing
+				`	"Sorry, I'm not interested."`
+			`	Andre looks disappointed, but accepting. "In which case, the Navy thanks you for your time, and we'll just end this here." He escorts you back through the checkpoints to the main spaceport, and bids you farewell.`
+				decline
+			label briefing
+			`	Andre looks please. "Excellent. For this next part, it is probably easiest to just show you." He gets up, and leads you down the stairs to a large hangar. Flipping a switch, the banks of light start to flicker on, revealing the largest turret you have ever seen in human space.`
+			scene "scene/broadsiderscene"
+			`	"This is the NAT-420mm Artillery, colloquially known as 'The Broadsider.' It fires 420mm slugs at 1.5 times ten to the 4th meters per second, or roughly 0.005% the speed of light. It has a rate of fire of a slug per second, and has sufficient density to remain sufficiently cohesive through several AU's worth of dust and micro-asteroids." He beams with pride as he waves you forward to take a closer look at it. "The slugs themselves rely purely on kinetic mass to damage their target, but my team did manage to develop a interesting little feature: We figured out a way to have it generate a micro-field that destabilizes standard shields, enabling subsequent shots to penetrate through to the hull itself. It is a neat trick, although some of my team still think we should have just added in some explosives or guidance systems instead."`
+			`	He stares at the turret for a moment, then looks back to you. "Now, the consulting job we have for you is fairly simple: We have a carrier equipped with these ready to launch for a demonstration test. We would like you to escort it to the test site, and observe as it destroys the target, then provide your opinion on how useful this is likely to be against future Sestor invasions." He pauses, glances at something on his commlink, and then looks back to you. "Oh, and we have a group of VIPs who are also tagging along to observe the test. If you could keep them safe, it would be much appreciated."`
```

Comment:
```suggestion
			`	He stares at the turret for a moment, then looks back to you. "Now, the consulting job we have for you is fairly simple: we have a carrier equipped with these ready to launch for a demonstration test. We would like you to escort it to the test site, and observe as it destroys the target, then provide your opinion on how useful this is likely to be against future Sestor invasions." He pauses, glances at something on his commlink, and then looks back to you. "Oh, and we have a group of VIPs who are also tagging along to observe the test. If you could keep them safe, it would be much appreciated."`
```